lib/tapioca/gem/pipeline.rb

```diff
@@ -355,7 +355,7 @@ def defined_in_gem?(constant, strict: true)
       def get_file_candidates(constant)
         wrapped_module = Pry::WrappedModule.new(constant)
 
-        wrapped_module.candidates.map(&:file).to_a.compact
+        wrapped_module.send(:method_candidates).flatten.filter_map(&:source_file).uniq
```

Comment:
I think the PR description should be copied here as comment.

Reply:
But if it is in a PR description and commit message already, can we find it using `git blame`? Do we really need a comment as well for that?